src/renderPage.node.ts

```diff
@@ -677,7 +677,7 @@ function isFileRequest(urlNormalized: string) {
 }
 
 async function render500Page(
-  err: unknown,
+  err: Error,
```

Comment:
It actually is `unknown` because the user may have `throw 'some-error'` instead of `throw new Error('some-error')`